keyboards/gmmk/pro/ansi/keymaps/jqs/config.h

```diff
@@ -0,0 +1,7 @@
+#define RGB_MATRIX_KEYPRESSES
```

Comment:
This needs an include guard (`#pragma once`), and
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15